src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -72,7 +72,7 @@ await transactionOperationContext.ReadForMemoryAsync(HttpContext.Request.Body, "
                         await WriteServerInfo(archive, jsonOperationContext, localEndpointClient);
                         foreach (var databaseName in requestHeader.DatabaseNames)
                         {
-                            WriteDatabaseRecord(archive, databaseName, jsonOperationContext, transactionOperationContext);
+                            await WriteDatabaseRecord(archive, databaseName, jsonOperationContext, transactionOperationContext);
```

Comment:
Not doing `await` here can cause access violation exception, heap corruption. Need to note that for possible scenarios. 
I checked `v5.2`, and it doesn't return void.